README.md

```diff
@@ -32,7 +32,18 @@ brew upgrade pscale
 
 #### Windows
 
-`pscale` is available as downloadable binary from the [releases](https://github.com/planetscale/cli/releases/latest) page.
+On Windows, `pscale` is available via [scoop](https://scoop.sh/), and as a downloadable binarty from the [releases](https://github.com/planetscale/cli/releases/latest) page:
+
+```
+scoop bucket add pscale https://github.com/planetscale/scoop-bucket.git
+scoop install pscale
```

Comment:
should we add `scoop install pscale mysql` here so that people get the CLI as well? I can test that out if we're interested.

Reply:
I like that idea! If scoop supports it, it would mean that we can make sure the user also installs MySQL :)

Reply:
cool - let's get this released and then I can test it out on a Windows VM and update the docs.